src/engraving/libmscore/instrtemplate.h

```diff
@@ -157,12 +157,12 @@ struct InstrumentGroup {
 //---------------------------------------------------------
 
 struct InstrumentIndex {
-    int groupIndex;
-    int instrIndex;
+    int groupIndex = 0;
+    int instrIndex = 0;
+    int templateCount = 0;
     InstrumentTemplate* instrTemplate;
```

Comment:
It would be nice to set instrTemplate to nullptr by default

Reply:
Done.